os/booting-on-vagrant.md

```diff
@@ -50,20 +50,56 @@ coreos:
     listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
     listen-peer-urls: http://$private_ipv4:2380
   fleet:
-      public-ip: $public_ipv4
+    public-ip: $public_ipv4
+  flannel:
+    interface: $public_ipv4
   units:
     - name: etcd2.service
       command: start
     - name: fleet.service
       command: start
+    - name: flanneld.service
+      drop-ins:
+      - name: 50-network-config.conf
+        content: |
+          [Service]
+          ExecStartPre=/usr/bin/etcdctl set /coreos.com/network/config '{ "Network": "10.1.0.0/16" }'
+      command: start
 ```
 
 The `$private_ipv4` and `$public_ipv4` substitution variables are fully supported in cloud-config on Vagrant. They will map to the first statically defined private and public networks defined in the Vagrantfile.
 
+Also please note that you should not put your public SSH keys in cloud-config as Vagrant will overwrite them with its own automatically generated key.
```

Comment:
Let's rewrite this to not start with "also". What about:

"There is no need to add an SSH key since Vagrant will automatically generate and use it's own SSH key. Any keys added will be overwritten."